articles/custom_storage_location.md

```diff
@@ -1,57 +1,97 @@
 ---
 title: "Custom Storage Locations"
 layout: article
-excerpt: Follow these steps to set up an external S3 bucket and access it with Synapse. 
+excerpt: Follow these steps to set up custom storage locations and access them with Synapse. 
 category: howto
 ---
 
-## **Creating a download bucket**
+<style>
+#image {
+    width: 100%;
+}
+#imageSmall {
+    width: 40%;
+}
+</style>
 
-## Steps for adding an external S3 bucket
+# Overview
+One of the main features of Synapse is to act as a repository for scientific data. Access to all data in Synapse is controlled with the use of [Sharing settings](/articles/access_controls). The Sharing setting is often the only required control on non-human subjects data whereas human subjects data requires additional controls. While Synapse provides physical storage for files (using Amazon's S3), not all data 'in' Synapse is stored on Synapse controlled locations. For example, data files can physically reside on a user owned S3 buckets, SFTP servers, or proxy servers. Creating a custom storage location allows users ownership and control of their files, especially in cases where there is a large amount of data or cases where there are additional restrictions that need to be set on the data.
 
-### Share your bucket with Synapse
-First step is to allow Synapse to access your s3 bucket. You can choose to allow read-only access or read-write access. Either way, you have to make sure that objects that are referenced or managed by Synapse are not modified or deleted.
+## Using SFTP
 
-#### 1. Go to **[Amazon Web Services S3 console](https://console.aws.amazon.com/s3)** and login under your account
+To set the storage location as your SFTP server:
```

Comment:
A quick summary of what we are doing.

To setup an sftp as a storage location we need to change the settings on the project.  Specifically we set the storageLocation.  This is best done using either R or Python but has alpha support in the web browser.